kserve/README.md

```diff
@@ -21,6 +21,32 @@ bash scripts/setup.sh
 
 > *Note: It may be necessary to run with `root` privileges if the default `/raid/nim` is not accessible.
 
+`NOTE:` If you have StorageClass and plan to use that please follow the below steps, please make sure run the below steps before run `bash scripts/setup.sh`
```

Comment:
Trying to use the impersonal tense in these docs. Please re-write to avoid using pronouns such as "you".

Reply:
done